docs/Options.md

```diff
@@ -523,6 +523,7 @@ This section show only when there are active jobs in the background.
 | `SPACESHIP_JOBS_SUFFIX` | ` ` | Suffix after the jobs indicator |
 | `SPACESHIP_JOBS_SYMBOL` | `✦` | Character to be shown when jobs are hiding |
 | `SPACESHIP_JOBS_COLOR` | `blue` | Color of background jobs section |
+| `SPACESHIP_JOBS_SHOW_AMOUNT_ALWAYS` | `false` | Show number of jobs (`true`, `false`) |
```

Comment:
`_SHOW_AMOUNT` with values of `true | false | always` is more extensible and in line with other scripts, like you had in #459 :)